src/apply-preserving-inline-style.js

```diff
@@ -222,11 +222,13 @@
 
   scope.apply = function(element, property, value) {
     ensureStyleIsPatched(element);
-    element.style._set(scope.propertyName(property), value);
+    if (typeof scope.propertyName == 'function') {
```

Comment:
100% agree. Douglas, are you able to shred some lights on where i can start the investigation? the way i used animation is below:

typescript part:

@Component({
    animations: [
        trigger(
            'defAnimation', defaultComponentAnimation
        )
    ],
    changeDetection: ChangeDetectionStrategy.OnPush,
    selector: 'xxx',
    styleUrls: ['xxx.component.scss'],
    templateUrl: './xxx.component.html'
})

export let defaultComponentAnimation: AnimationMetadata[] = [
    transition(
        ':enter', [
            style({ opacity: 0}),
            animate('400ms', style({ opacity: 1}))
        ]
    )
];

really appreciate for your help.